Assets/Scripts/Inventory/PickupItem.cs

```diff
@@ -2,31 +2,34 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-[RequireComponent(typeof(Item))]
-public class PickupItem : MonoBehaviour
+namespace FunMath
 {
-    [SerializeField]
-    private Item item;
-    void Start()
+    [RequireComponent(typeof(Item))]
+    public class PickupItem : MonoBehaviour

```

Comment:
I still think that this `PickupItem` should be a part of the `PlayerController`.

Reply:
That would be up to the PlayerController to have changes, which I'm not in charge of.